docs/explanation/package-model.rst

```diff
@@ -263,4 +263,4 @@ Resources
 .. _debhelper package: https://launchpad.net/ubuntu/+source/debhelper
 .. _bash Software: https://www.gnu.org/software/bash/
 .. _Debian bash package: https://tracker.debian.org/pkg/bash
-.. _Ubuntu bash package: https://launchpad.net/ubuntu/+source/bash
\ No newline at end of file
+.. _Ubuntu bash package: https://launchpad.net/ubuntu/+source/bash
```

Comment:
question: What is the change here?

Reply:
I'd say the line ending, but it appears it was neither removed, nor changed, so I don't know 🤷